packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -57,6 +38,8 @@ const IDVForm = ({
         sample_image: '',
     };
 
+    const IDV_NOT_APPLICABLE_OPTION = React.useMemo(() => getIDVNotApplicableOption(), []);
```

Comment:
```suggestion
    const IDV_NOT_APPLICABLE_OPTION = React.useMemo(() => getIDVNotApplicableOption(), []);
```

@likhith-deriv, @yauheni-deriv, is it necessary to cache this?

as I checked, this is it's definition `export const getIDVNotApplicableOption = () => ({ id: '#NA', text: localize('I want to do this later') });`, so I don't think it requires caching. Am I missing something? 🤔